src/lib/components/submit/SubmitReviewAddNodeProvider.svelte

```diff
@@ -0,0 +1,25 @@
+<!-- ProviderDetails.svelte -->
+<script lang="ts">
+	import { fade } from 'svelte/transition';
+	import Container from '$lib/components/ui/Container.svelte';
+	import type { ProposalEditableMetadata } from '$lib/types/juno';
+
+	export let metadata: ProposalEditableMetadata | undefined;
+</script>
+
+<div in:fade>
```

Comment:
this fade effect has no effect given that svelte requires using the tag within a if. I'm surprised the linter does not complain.